tests/pkg/notifications/asserters/notifications.go

```diff
@@ -83,6 +89,7 @@ func assertNotificationsCount(t *testing.T, body []byte, objectID string, count
 	}
 }
 
+// will be used one the test that depend on the items structure are adapted to the new test format
```

Comment:
Yes it will be used in the futer as the coment stated. Once the test that depends on the items structure is migrated to the new format the function will be used in the asserter or new asserter will be created if needed.